crates/core_arch/src/aarch64/neon/mod.rs

```diff
@@ -749,6 +749,22 @@ pub unsafe fn vld1q_p16(ptr: *const p16) -> poly16x8_t {
     ))
 }
 
+/// Load multiple single-element structures to one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon,aes")]
+#[cfg_attr(test, assert_instr(ldr))]
+pub unsafe fn vld1_p64(ptr: *const p64) -> poly64x1_t {
+    transmute(u64x1::new(*ptr))
+}
+
+/// Load multiple single-element structures to one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon,aes")]
+#[cfg_attr(test, assert_instr(ldr))]
+pub unsafe fn vld1q_p64(ptr: *const p64) -> poly64x2_t {
+    transmute(u64x2::new(*ptr, *ptr.offset(1)))
```

Comment:
There was a concern raised in #1148 that this doesn't get optimized down to a single instruction.

Reply:
I agree, all others have been already changed to use `read_unaligned()` instead which cannot be seen in the diff as it this PR is based on an older commit.

Reply:
That makes sense.